uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-actions.component.js

```diff
@@ -22,13 +23,16 @@ define([
       return sharedSettings.sharedPathTpl + 'components/leave-request-actions.html';
     }],
     controllerAs: 'actions',
-    controller: ['$log', '$q', '$rootScope', 'OptionGroup', 'dialog', 'shared-settings', controller]
+    controller: LeaveRequestActionsController
   });
 
-  function controller ($log, $q, $rootScope, OptionGroup, dialog, sharedSettings) {
+  LeaveRequestActionsController.$inject = ['$log', '$rootScope', 'dialog', 'pubSub', 'shared-settings'];
```

Comment:
I am not sure if I should do it because I have not made any changes to the service itself, I have just used it. @igorpavlov @AkA84 Let me know what you think.